lua/entities/gmod_wire_hologram.lua

```diff
@@ -4,15 +4,31 @@ ENT.PrintName = "Wire Hologram"
 ENT.RenderGroup = RENDERGROUP_OPAQUE
 ENT.DisableDuplicator = true
 
-function ENT:SetPlayer(ply)
-	self:SetVar("Founder", ply)
-	self:SetVar("FounderIndex", ply:UniqueID())
-
-	self:SetNWString("FounderName", ply:Nick())
+function ENT:SetupDataTables()
+	self:NetworkVar( "Entity", 0, "PlayerEnt" )
 end
 
 function ENT:GetPlayer()
-	return self:GetVar("Founder", NULL)
+	local ply = self:GetPlayerEnt()
+
+	if self.steamid == "" then
+		if ply:IsValid() then
+			self.steamid = ply:SteamID()
+		end
+	elseif SERVER and not ply:IsValid() then
+		local newply = player.GetBySteamID(self.steamid)
+		if newply then
+			self:SetPlayer(newply)
+			ply = newply
+		end
+	end
```

Comment:
This needs to be replaced with a join hook that finds holograms with invalid owner and sets them if the id matches.